spec.html

```diff
@@ -26488,7 +25947,6 @@ <h1>Boolean.prototype</h1>
       <h1>Properties of the Boolean Prototype Object</h1>
       <p>The Boolean prototype object:</p>
       <ul>
-        <li>is the intrinsic object <dfn>%BooleanPrototype%</dfn>.</li>
```

Comment:
How is it implicit at all? It's the object on `Boolean.prototype`, `Boolean` is `%Boolean%`.

Reply:
Sorry, when you say:
> It's the object on `Boolean.prototype`

you mean that the phrase "the Boolean prototype object" denotes (the initial value of) `Boolean.prototype`, right?

I agree that that's the intent, but this PR doesn't say so. I'm not saying people won't figure it out, I'm saying there's no reason it should be something they need to figure out.

Reply:
I'm saying that `%Boolean.prototype%` is already in the spec as "the original value of the `prototype` property on `%Boolean%`" - so perhaps just the sentence [here](https://tc39.es/ecma262/#sec-boolean.prototype) needs to be updated, since it's currently a tautology?

Reply:
> I'm saying that `%Boolean.prototype%` is already in the spec as "the original value of the `prototype` property on `%Boolean%`"

Oh, okay. Well, I agree that that's in the spec, but I don't see how that answers my original point.

> - so perhaps just the sentence [here](https://tc39.es/ecma262/#sec-boolean.prototype) needs to be updated, since it's currently a tautology?

I agree that's currently a tautology. How would you update it? Maybe just demote it to a Note?

Reply:
I don't think it needs even a note; just making sure that "the Boolean Prototype object" is also noted somewhere as "the value of `.prototype` on `%Boolean%` should suffice?

Reply:
> just making sure that "the Boolean Prototype object" is also noted somewhere as "the value of `.prototype` on `%Boolean%` should suffice?

Okay, I'm not sure how we got here, but yes, this aligns with two of the solutions I originally suggested.

And the third solution was to eliminate the phrase "the Boolean Prototype object". Does the spec benefit from introducing that phrase, rather than just saying "%Boolean.prototype%" (or `Boolean.prototype`, suitably qualified)?